### DISCUSSION TITLE
HTML font tag

### DISCUSSION THREAD

**Discussion by crairdin** [ueberdosis/tiptap#2579] [General Questions]
I inherited tiptap in my project so I neither chose it nor understand it. We were using version 1 when it was abandoned and replaced with version 2 beta. We had an extension that supported the old HTML <font> tag. This is essential for us, as we're using tiptap as a note editor in our app, and users have lots of existing notes that use the font tag, and using spans and css may or may not be supported in our app when running on other platforms. So we're stuck with <font>.

It's not clear to me how to port the old extension to version 2. I'm sure this is thoroughly documented, but I don't see it. If someone can point me to clear, step-by-step documentation for porting old extensions to version 2, it would help me out significantly.

**Comment by philippkuehn** [comment]
I think there is a good guide how to create custom nodes: https://tiptap.dev/guide/custom-extensions

**Reply by crairdin** [reply]
That is the guide for creating custom nodes. I was trying to minimize what I had to learn about writing custom nodes and just apply some rules to port the old extensions. I solved the problem by removing tiptap from my project. :-)
